pages/Cart/components/CouponField/components/Layout/style.js

```diff
@@ -34,6 +34,7 @@ const input = css({
   top: 0,
   outline: 0,
   bottom: 6,
+  padding: 0,
```

Comment:
The cursor is still in a wrong position if you type something und delete it again.

Reply:
I've asked our PO and we'll keep it like that right now